control/control.go

```diff
@@ -76,7 +76,8 @@ func (s *SourceParagraph) Maintainers() []string {
 // information that will be eventually put lovingly into the .deb file
 // after it's built on a given Arch.
 type BinaryParagraph struct {
-	Paragraph
+	rfc2822.Paragraph
+
 	Architectures []dependency.Arch `control:"Architecture"`
```

Comment:
This tag is a perfect example of the "choice" I mentioned -- `control:"Architecture"` should _probably_ be converted to `rfc2822:"Architecture"` if we want to make this new library "pure".